### ISSUE TITLE
Close all existing dependabot pull request.

### ISSUE THREAD

**Issue by sulenchy** [NoroffFEU/edu-gate#137]
We want to resolve the vulnerability ourselves, so all pull request labeled with "bot" needs to be closed . [PULL REQUESTS](https://github.com/NoroffFEU/edu-gate/pulls)

**Comment by Anne-Zwift** [comment]
@kristinatonnessen Hi I can't understand if there is something to do here?

**Comment by kristinatonnessen** [comment]
@Anne-Zwift yes, here is the links to the pull requests, the once with "bot" labels needs to be closed 😃 https://github.com/NoroffFEU/edu-gate/pulls

**Comment by kristinatonnessen** [comment]
Closed all Dependabot pull requests, just waiting on Abi to answer if we should close the last pull request he opened, and if we should remove all the Depedabot alerts as well? @sulenchy

**Comment by kristinatonnessen** [comment]
@Janicke92 ready to be moved with your approval, heard back from Abi and we should leave it as it is now 😃
